vsphere/datadog_checks/vsphere/data/conf.yaml.example

```diff
@@ -138,6 +138,16 @@ instances:
     #   cluster:
     #     - <CLUSTER_REGEX>            # Only possible with "collection_type: historical"
 
+    ## @param collect_tags - boolean - optional - default: false
+    ## If true, the vSphere Tags will be collected.
+    ## The final tag sent to Datadog is composed of the vSphere tag category as key and vSphere tag name as value.
+    ##    <TAG_CATEGORY>:<TAG_NAME>
+    ## Examples:
+    ## - os_type:windows
+    ## - application_name:my_app
+    #
+    # collect_tags: false
```

Comment:
To avoid collision. Should we add another config `tag_prefix`, that default to `vsphere_` ?

Tag will then look like `<TAG_PREFIX>_<TAG_CATEGORY>:<TAG_NAME>`

e.g. 1 `vsphere_os_type:windows`
e.g. 2 `vsphere_application_name:my_app`

Reply:
This feature has been added.